### DISCUSSION TITLE
Latest release (24.07)

### DISCUSSION THREAD

**Discussion by gj1118** [helix-editor/helix#11180] [General]
Hi, 
Not an issue, per se, but I am not seeing the changelog for the latest release (24.07). Also the latest release is not published on homebrew.

Is this something that I am only experiencing? If so that is my bad. 

Thanks

**Comment by gabydd** [comment]
Hi there, change log is here https://github.com/helix-editor/helix/blob/master/CHANGELOG.md homebrew is not packaged by us so that is dependent on when someone makes a pr to them and they accept it

**Reply by gabydd** [reply]
you can track the homebrew pr here: https://github.com/Homebrew/homebrew-core/pull/177333

**Reply by the-mikedavis** [reply]
I've just edited the 24.07 release page to link out to the changelog
